src/toast/ops/filterbin.py

```diff
@@ -114,7 +113,7 @@ def normalize(self, good=None):
         """Normalize templates"""
         for start, stop, template in zip(self.starts, self.stops, self.templates):
             if good is None:
-                norm = np.sum(template**2) ** 0.5
+                norm = np.sum(template ** 2) ** 0.5
```

Comment:
This is old `black` isn't it? Or did they change the unary operator spacing back?

Reply:
Sorry, ran with a stale black, just pushed the fix.